src/game_state.rs

```diff
@@ -0,0 +1,48 @@
+use super::messenger::{MessengerOperations, SendPacketMessage};
+use super::packet;
+use super::packet::Packet;
+use std::sync::mpsc::Sender;
+
+// Called when client pings the server
+pub fn play(p: Packet, conn_id: i32, messenger: Sender<MessengerOperations>) {
+    println!("Switched to play state");
+}
+
```

Comment:
I put this in slack but I don't really understand this file- it doesn't hold any state and only seems to hold stuff relevant to login right now